db/memtable.h

```diff
@@ -409,7 +409,7 @@ class MemTable {
   ConcurrentArena arena_;
   std::unique_ptr<MemTableRep> table_;
   std::unique_ptr<MemTableRep> range_del_table_;
-  bool is_range_del_table_empty_;
+  std::atomic_bool is_range_del_table_empty_;
```

Comment:
looks like we use std::atomic<bool> much more often in the codebase, any idea if there is any difference?

Reply:
It's typedef'd to `std::atomic<bool>` (see "Type Aliases" section on https://en.cppreference.com/w/cpp/atomic/atomic). I can change it for consistency if you want though.